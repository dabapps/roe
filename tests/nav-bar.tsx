import * as React from 'react';
import * as renderer from 'react-test-renderer';
import NavBar from '../src/ts/components/navigation/nav-bar';

describe('NavBar', () => {

  it('should match snapshot', () => {
    const tree = renderer.create(
      <NavBar />
    );

    expect(tree).toMatchSnapshot();
  });

  it('should take regular element attributes', () => {
    const tree = renderer.create(
      <NavBar className="my-class" />
    );

    expect(tree).toMatchSnapshot();
  });

});
