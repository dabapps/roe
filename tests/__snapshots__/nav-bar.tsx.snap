// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match snapshot 1`] = `
<div
  className="nav-bar"
/>
`;

exports[`NavBar should take regular element attributes 1`] = `
<div
  className="nav-bar my-class"
/>
`;
